models/Recording.js

```diff
@@ -228,7 +228,7 @@ module.exports = function(sequelize, DataTypes) {
   };
 
   var tagOfType = (tagWhats, tagTypeSql) => {
-    return 'EXISTS (' + recordingTaggedWith(tagWhats, tagTypeSql) + ') OR EXISTS(' + trackTaggedWith(tagWhats, tagTypeSql) + ')';
+    return '(EXISTS (' + recordingTaggedWith(tagWhats, tagTypeSql) + ') OR EXISTS(' + trackTaggedWith(tagWhats, tagTypeSql) + '))';
   };
 
   var notTagOfType = (tagWhats, tagTypeSql) => {
```

Comment:
Should this be enclodes in brakcs as well?
```    return 'NOT EXISTS (' + recordingTaggedWith(tagWhats, tagTypeSql) + ') AND NOT EXISTS(' + trackTaggedWith(tagWhats, tagTypeSql) + ')';```

Reply:
This one doesn't 'need brackets' its the 'or' clause that causes the problem.